portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,110 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
+
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+
   </head>
   <body>
-    <div id="content">
-      <h1>Kayla Leung</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/kaylaleung.jpeg"/>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <!-- Toggle Side View -->
+    <input type="checkbox" name="">
+    <span class="icon"></span>
+    
+    <!-- Left Side Bar -->
+    <div class="sidebar">
+
+        <img src="images/kaylaleung.jpeg" style="width:175px;height:175px;" alt="user profile photo">
```

Comment:
As mentioned before, prefer to use the width/height properties over CSS, much less inline CSS.

For alt text, think about what you would want a screen reader to say. Since this is a photo of you, I'd expect it to say something like "Photo of Kayla Leung".

Reply:
Ah ok, I previously misunderstood what you had meant by properties of the image vs CSS properties. Fixed now!